eng/scripts/dump_process.ps1

```diff
@@ -0,0 +1,6 @@
+Set-Location $args[0]
+
+while ($true) {
+    Get-Process > artifacts/log/processes/runningProcesses.$(get-date -f yyyy-MM-dd-HH-mm-ss).txt
```

Comment:
We don't really need anything but the last copy of this file and our artifacts bundles are already unreadable. Repeatedly overwrite the same file, append to a single file, or delete all but the last one or two copies of this file. (The specific choice doesn't matter to me.)